akka-docs/src/main/paradox/persistence.md

```diff
@@ -819,6 +819,170 @@ For more advanced schema evolution techniques refer to the @ref:[Persistence - S
 
 ## Testing
 
+### Persistence TestKit
+
+There is the persistence testkit for testing persistent actors. 
+It allows to check messages saved in storage, emulate storage exceptions and behavior.
+
+To use the testkit you need to add the following dependency in your project:
+
+@@dependency[sbt,Maven,Gradle] {
+  group="com.typesafe.akka"
+  artifact="akka-persistence-testkit_$scala.binary_version$"
+  version="$akka.version$"
+}
+
+There are two testkit classes which have similar api:
+
+ * @scala[`PersistenceTestKit` class]@java[`PersistenceTestKit` class] is for messages
+ * @scala[`SnapshotTestKit` class]@java[`SnapshotTestKit` class] is for snapshots
```

Comment:
since same names, no need to use the `@scala` and `@java`

Reply:
corrected